src/main/scala/coreplex/Configs.scala

```diff
@@ -22,7 +22,7 @@ class BaseCoreplexConfig extends Config ((site, here, up) => {
   case BuildCore => (p: Parameters) => new Rocket()(p)
   case RocketCrossing => Synchronous
   case RocketTilesKey =>  Nil
-  case DMKey => new DefaultDebugModuleConfig(site(XLen))
+  case DMKey => DefaultDebugModuleConfig(site(XLen))
```

Comment:
i'd use named parameters here `DefaultDebugModuleConfig(nDMIAddrSize = site(XLen))` only because this class takes so many Ints in a row and you might want to swap their order later